cmd/trader/trader.go

```diff
@@ -34,146 +41,232 @@ func main() {
 	// Parse the option parameters
 	numberOfOrders := flag.Int("order", 10, "number of orders")
 	timeInterval := flag.Int("time", 15, "time interval in second")
+	cancelRequest := flag.Bool("cancel", false, "cancel request")
 
-	// Get nodes/darkPool details
-	multiAddresses := getNodesDetails()
-	nodes := make([]identity.MultiAddress, len(multiAddresses))
-	for i := 0; i < len(multiAddresses); i++ {
-		multi, err := identity.NewMultiAddressFromString(multiAddresses[i])
-		if err != nil {
-			log.Fatal(err)
-		}
-		nodes[i] = multi
-		log.Println(base58.Encode(nodes[i].ID()))
-	}
+	// Logger
+	logs, err := logger.NewLogger(logger.Options{
+		Plugins: []logger.PluginOptions{
+			logger.PluginOptions{
+				File: &logger.FilePluginOptions{
+					Path: "stdout",
+				},
+			},
+		}})
 
-	// Create a trader address
+	// Create a test trader address
 	address, _, err := identity.NewAddress()
 	if err != nil {
-		log.Fatal(err)
+		panic(err)
```

Comment:
In general, `log.Fatal` is preferable over `panic`. A panic is more commonly used when there is still a chance of recovery.

Reply:
Okay, thanks. will change that.